core/src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -253,36 +183,13 @@ public static ClusterHealthResponse readResponseFrom(StreamInput in) throws IOEx
     public void readFrom(StreamInput in) throws IOException {
         super.readFrom(in);
         clusterName = in.readString();
-        activePrimaryShards = in.readVInt();
```

Comment:
@bleskes: The current implementation should already be wire compatible or am I missing something? I took care that `ClusterStateHealth` writes / reads all fields that I've moved in the exactly same order as before.